0.4/Dockerfile

```diff
@@ -32,10 +32,13 @@ ENV JANUS_VERSION=${JANUS_VERSION} \
     janusgraph.storage.directory=/var/lib/janusgraph/data \
     gremlinserver.graph=/etc/opt/janusgraph/janusgraph.properties \
     gremlinserver.threadPoolWorker=1 \
-    gremlinserver.gremlinPool=8
+    gremlinserver.gremlinPool=8 \
+    DEBIAN_FRONTEND=noninteractive
```

Comment:
Why was this change necessary? [The Docker docs recommend](https://docs.docker.com/engine/faq/#why-is-debian_frontendnoninteractive-discouraged-in-dockerfiles) against using this.

Reply:
Removed from ENV and added it to specific command as if this `DEBIAN_FRONTEND=noninteractive` not provided then installation of krb5-user prompts kerberos related configs like realm.

Reply:
OK, good to know and the solution you used now here sounds good to me.